include/robot_fingers/n_joint_blmc_robot_driver.hxx

```diff
@@ -114,6 +115,7 @@ typename NJBRD::Config NJBRD::Config::load_config(
 
     set_config_value(user_config, "max_current_A", &config.max_current_A);
     set_config_value(user_config, "has_endstop", &config.has_endstop);
+    set_config_value(user_config, "homing_at_endstop", &config.homing_at_endstop);
```

Comment:
This will make it a mandatory option.  For backward-compatibility with existing config files, it would be better to make it optional by wrapping it with a `if (user_config["homing_at_endstop"])`-check.